lib/route_translator/host_path_consistency_lambdas.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module RouteTranslator
+  module HostPathConsistencyLambdas
+    module_function
+
+    def for_locale(locale)
+      locale = sanitize_locale(locale)
```

Comment:
Please avoid the function call and reassign locale.

Since `.to_s.gsub('native_', '')` is called three times, I will then provide a global method to sanitize the locale